pkg/netpol/eval/internal/k8s/namespace.go

```diff
@@ -39,7 +39,8 @@ func NamespaceFromCoreObject(ns *corev1.Namespace) (*Namespace, error) {
 	for k, v := range ns.Labels {
 		n.Labels[k] = v
 	}
-
+	// @todo/tbd : should also add the name label as "name:<val>"  or assume policy rules
```

Comment:
I think that `name` key is not auto generated, as opposed to `kubernetes.io/metadata.name`.. please verify. If that is the case, should be able to rely on the labels from the namespace resource itself?

Reply:
true 
according to [this](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#automatic-labelling) ;  only `kubernetes.io/metadata.name` is automatically generated for namespaces.

and according to [this](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-metadata-name) it is recommended to use `kubernetes.io/metadata.name` as the label selector of name;

so if we have a just `name` key in the rule-selector; we'll need to compare with the actual ns name ?

Reply:
> so if we have a just name key in the rule-selector; we'll need to compare with the actual ns name

no, should assume that such key should be defined in the labels of the namespace resource